packages/config_h/config_h.v0.12.0/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
```

Comment:
This package has a fairly generic name -- if it's intended to be more internal, perhaps preface it with `js-config`?